connection_test.go

```diff
@@ -284,7 +284,7 @@ func TestBadRequest(t *testing.T) {
 		assert.Equal(t, ErrCodeBadRequest, GetSystemErrorCode(err))
 
 		calls := relay.NewMockStats()
-		calls.Add(ts.ServiceName(), ts.ServiceName(), "Noone").Failed("bad-request").End()
+		calls.Add(ts.ServiceName(), ts.ServiceName(), "Noone").Fluent().Failed("bad-request").End()
```

Comment:
The `Add` method is not part of any relay interface, we only have a `Begin` which takes a call frame. Am I missing something?